build.sbt

```diff
@@ -20,22 +20,23 @@ scalacOptions ++= Seq(
   "-Wconf:src=routes/.*:s"
 )
 
-val bootstrapVersion = "7.9.0"
+val bootstrapVersion = "8.4.0"
+val playVersion = "play-30"
 
 libraryDependencies  ++= Seq(
```

Comment:
I'd split into `compileDependencies` and `testDependencies` to make it clearer. Mapping the second set of dependences over `.map(_ % "test, it")`